src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -19,6 +20,12 @@ export default class OrganizationIntegrations extends AsyncComponent {
   reloadOnVisible = true;
   shouldReloadOnVisible = true;
 
+  componentDidMount() {
+    analytics('integrations.index_viewed', {
+      org_id: this.props.params.orgId,
```

Comment:
This stores the slug not the numeric ID, that's wrong.

Reply:
So we should change it then?

Reply:
Yes, was just leaving this as a note for myself.